wiki/People/The_Team/Beatmap_Nominators/en.md

```diff
@@ -123,7 +124,9 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 
 | Name | Additional languages |
 | :-- | :-- |
+| ![][flag_CL] [-Joni-](https://osu.ppy.sh/users/9988837) | Spanish |
```

Comment:
shouldn't -Joni- be added to Prob BN table instead?

Reply:
yep my mistake, thanks!